wundergraph_cli/src/print_schema/mod.rs

```diff
@@ -2,6 +2,7 @@ use crate::database::InferConnection;
 use crate::infer_schema_internals::*;
 use std::error::Error;
 use std::io::Write;
+use std::thread;
```

Comment:
Seems like this import is not needed?

Reply:
Indeed, the import is removed in 375c03aea376cb057d3261145ee1344ed076fba5